### PR TITLE
MAINT: fix a couple of trans._.

### PR DESCRIPTION
**Pull request by Carreau**
Also remove a f-string from within trans, or it does not make sens

### CODE REVIEW

File: napari/_qt/qt_main_window.py [hunk 1]
```diff
@@ -1235,7 +1235,10 @@ def _screenshot(
             if size is not None:
                 if len(size) != 2:
                     raise ValueError(
-                        f'screenshot size must be 2 values, got {len(size)}'
+                        trans._(
+                            'screenshot size must be 2 values, got {len_size}',
+                            len_size=len(size),
+                        )
                     )
```
Comment (Czaki) [inline napari/_qt/qt_main_window.py:1242]: this is a pure API exception, I'm not sure if it should be translated, as it should not be triggered by GUI interaction. But maybe...

File: napari/_qt/qt_viewer.py [hunk 2]
```diff
@@ -626,8 +626,12 @@ def _save_layers_dialog(self, selected=False):
...
         )
         logging.debug(
-            f'QFileDialog - filename: {filename or None} '
-            f'selected_filter: {selected_filter or None}'
+            trans._(
+                'QFileDialog - filename: {filename} '
+                'selected_filter: {selected_filter}',
+                filename=filename or None,
+                selected_filter=selected_filter or None,
+            )
```
Comment (Czaki) [inline napari/_qt/qt_viewer.py:634]: If I correctly remember the trans should consume only single-line strings. Also, I'm not sure if debug messages should be translated.

**Review by Czaki — APPROVED** [review]
One string needs to be not translated or merged into a single line.

File: napari/_qt/qt_viewer.py (hunk 2 above)
Comment (Carreau) [reply napari/_qt/qt_viewer.py:634]: I don't think that is true for single line, I believe I fixed this already in #4588

File: napari/_qt/qt_main_window.py (hunk 1 above)
Comment (Carreau) [reply napari/_qt/qt_main_window.py:1242]: Yeah, maybe you are right. Just let me know and I can revert this.

File: napari/_qt/qt_viewer.py (hunk 2 above)
Comment (Carreau) [reply napari/_qt/qt_viewer.py:634]: WRT to if this has this to be translated, I don't really care. Happy to revert as well.

Comment (Czaki) [reply napari/_qt/qt_viewer.py:634]: #4588 should be more advertised. Why do I miss it.

Comment (Carreau) [reply napari/_qt/qt_viewer.py:634]: I think that's because not working on join strings was just a bug, and bug just get fixed :-)

**Review by liu-ziyang — APPROVED** [review]

**Pull request merged**